src/components/VErrorSection/VErrorImage.vue

```diff
@@ -51,6 +51,7 @@ export default defineComponent({
         let image = errorItem.image
         const errorImage: ErrorImage = {
           ...image,
+          originalTitle: image.title,
```

Comment:
I would've advocated for us to keep the original title as sent from the API in `title` (same as the field in the API) and store the transformed value in a different field. But I can see how that could have caused ripple effects across the codebase.

Reply:
That was my original thought, too, @dhruvkb !  But as you say, it would require a lot of changes in the codebase.